src/components/session/index.tsx

```diff
@@ -113,15 +113,15 @@ export const SessionPage: React.FC<{
                 {trace.timestamp.toLocaleString()}
               </div>
               <div className="mb-1 mt-2 text-xs text-gray-500">Scores</div>
-              <div className="flex flex-wrap content-start items-start gap-1">
+              <div className="group/scores flex flex-wrap content-start items-start gap-1">
                 <GroupedScoreBadges scores={trace.scores} />
+                <ExpertScoreButton
```

Comment:
<img width="654" alt="image" src="https://github.com/langfuse/langfuse/assets/17686849/4693cfab-9db5-435c-bd49-f9afd3036a70">
It would be great to not have the delete button, when there is only one selected score

Reply:
I just learned that the button will not only have an impact on the form, but will remove the score from the trace. This was not entirely clear to me when playing around initially.